src/image_intel.cpp

```diff
@@ -83,9 +83,9 @@ static MountPoints getMountPoints()
 
 static void unmountFilesystem(const std::string& filesystem)
 {
-    // Some systemd services may to occupate the RW partition
+    // Some systemd services may occupy the RW partition
     // and make delay up to 20 seconds. We should to wait them
```

Comment:
`resulting in a delay of up to 20 seconds. We should wait for them`

Reply:
done